tests/Symfony/Upgrade/Test/Fixer/FormGetnameToGetblockprefixFixerTest.php

```diff
@@ -12,10 +12,48 @@ public function testFix($expected, $input, $file)
         $this->makeTest($expected, $input, $file);
     }
 
+    /**
+     * @dataProvider provideFailingExamples
+     */
+    public function testWrongFix($expected, $input, $file, $unExpected)
+    {
+        $fixedCode = $this->getFixer()->fix($file, $input);
+        $this->assertNotSame($unExpected, $fixedCode);
+        $this->assertSame($expected, $fixedCode);
+    }
+
     public function provideExamples()
```

Comment:
@tobias-93 I see your point. So, I prefer to keep things simple. Please just remove this test cases and I will merge https://github.com/umpirsky/Symfony-Upgrade-Fixer/pull/19 as is, just without complicated tests. Sorry for being a pain. :)